third_party/nvfuser/test/test_loop_rotation.cpp

```diff
@@ -641,57 +597,51 @@ __global__ void CUDAGeneratedKernel(Tensor<float, 2> T0, Tensor<float, 2> T4) {
      = T1[0];
   #pragma unroll 1
   for(nvfuser_index_t i23 = 0; i23 < T0.size[0]; ++i23) {
-    int64_t i94;
-    i94 = 4 + i23;
-    int64_t i96;
-    i96 = 3 * (i94 % 5);
-    int64_t i118;
-    i118 = i116 + (T0.stride[0] * i23);
-    int64_t i161;
-    i161 = 1 + (3 * (i23 % 5));
-    int64_t i197;
-    i197 = 3 * i23;
-    bool b270;
-    b270 = i94 < T0.size[0];
-    bool b278;
-    b278 = i23 < T0.size[0];
+    int64_t i418;
+    i418 = 4 + i23;
+    int64_t i420;
+    i420 = 3 * (i418 % 5);
+    int64_t i504;
+    i504 = i502 + (T0.stride[0] * i23);
+    int64_t i663;
+    i663 = 1 + (3 * (i23 % 5));
+    int64_t i991;
+    i991 = 3 * i23;
+    bool b1549;
+    b1549 = i418 < T0.size[0];
     #pragma unroll
     for(nvfuser_index_t i21 = 0; i21 < 3; ++i21) {
-      T1[(i96 + i21)] = 0;
+      T1[(i420 + i21)] = 0;
     }
     NVFUSER_UPDATE_MAGIC_ZERO
     #pragma unroll
     for(nvfuser_index_t i21 = 0; i21 < 3; ++i21) {
-      int64_t i98;
-      i98 = i21 + nvfuser_zero;
-      if ((b270 && (i98 < 3))) {
-        T1[(i96 + i21)]
-           = T0[(i118 + (T0.stride[1] * i98))];
+      if (b1549) {
+        T1[(i420 + i21)]
+           = T0[(i504 + (T0.stride[1] * (i21 + nvfuser_zero)))];
       }
     }
     NVFUSER_UPDATE_MAGIC_ZERO
     #pragma unroll
     for(nvfuser_index_t i26 = 0; i26 < 2; ++i26) {
-      int64_t i189;
-      i189 = i26 + nvfuser_zero;
+      int64_t i923;
+      i923 = i26 + nvfuser_zero;
       T2[((1 + i26) % 2)]
-         = T1[(i161 + i26)];
+         = T1[(i663 + i26)];
       float T3[1];
       T3[0]
          = T2[(i26 % 2)];
-      if ((b278 && (i189 < 3))) {
-        T4[(i197 + i189)]
+      if ((i923 < 3)) {
```

Comment:
`i923 < 3` is trivial, we can not handle it currently. Will add support in next PR.